Core/Inc/iris_system.h

```diff
@@ -19,8 +19,8 @@
 /* USER CODE END Header */
 
 /* Define to prevent recursive inclusion -------------------------------------*/
-#ifndef __MAIN_H
-#define __MAIN_H
+#ifndef IRIS_SYSTEM_H

```

Comment:
There's an endif on line 101 that has an incorrect comment because of this change